src/BoE-Info.plist

```diff
@@ -151,21 +151,25 @@
 		</dict>
 	</array>
 	<key>CFBundleExecutable</key>
-	<string>Blades of Exile</string>
+	<string>$(EXECUTABLE_NAME)</string>
 	<key>CFBundleIconFile</key>
 	<string>BoE</string>
 	<key>CFBundleIdentifier</key>
-	<string>com.spidweb.bladesofexile</string>
+	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
```

Comment:
I'm not quite sure if it's a problem; I'd need to check. It's entirely possible that I'm actually hard-coding the domain as a string in the code, too, in which case this wouldn't really matter at all.